staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -1002,6 +1002,7 @@ var (
 	etpLocalNodePredicates []NodeConditionPredicate = []NodeConditionPredicate{
```

Comment:
`allNodePredicates` and `etpLocalNodePredicates` are identical now.  If I understand, we could just merge them, but since this is all replaced by `stableNodeSetPredicates` (which is beta) it's moot, and a smaller delta is preferable, right?

Reply:
Yes, this was the idea.